src/dependency-builder/build-tree.js

```diff
@@ -226,11 +228,13 @@ function groupMissing(missing, cwd, consumerPath, bindingPrefix) {
     return packageWithVersion ? Object.assign(foundPackages, packageWithVersion) :
                                 missingPackages.push(packageWithVersion);
   });
-
-
-  groups.packages = packageJson ? missingPackages.filter(packageName => !(packageName in packageJson.dependencies)) : missingPackages;
-  //groups.packages = missingPackages;
-
+  if (!R.isNil(dependencies)) {
```

Comment:
add to this condition the following:
`missingPackages && missingPackages.length`